README.rst

```diff
@@ -0,0 +1,37 @@
+This docker image is meant to be used to generate an allthethings.json file from Mozilla's
+Release Engineering Buildbot set up. Visit https://wiki.mozilla.org/ReleaseEngineering/How_To/allthethings.json to
+know more about the file and how to generate it.
+
+== Generate allthethings.json ==
+
+If you just want to generate allthethings.json locally run the following:
+
+.. code-block:: bash
+
+   docker pull armenzg/releng_buildbot_docker
+   # Everytime you run this it will do so from scratch
+   # XXX: Unfortunately piping to allthethings.json has info messages from the scripts
+   # TODO: Find way to grab file from container
+   docker run releng_buildbot_docker bash -c "\
```

Comment:
maybe add something like `cd /braindump && hg pull -u && ...` to make sure you are on the latest revision? The block below would rebuild the image and re-clone the repo using the latest revision, so it's not a requirement.